pixeltable/utils/parquet.py

```diff
@@ -109,18 +119,50 @@ def save_parquet(df: 'pixeltable.DataFrame', dest_path: Path, partition_size_byt
 
         _write_batch(current_value_batch, arrow_schema, temp_path / f'part-{batch_num:05d}.parquet')
 
-def get_part_metadata(path : Path) -> List[Tuple[str, int]]:
-    """
-        Args:
-            path: path to directory containing parquet files.
-        Returns:
-            A list of (file_name, num_rows) tuples for the parquet files in file name order.
-    """
-    parts = sorted([f for f in path.iterdir() if f.suffix == '.parquet'])
-    rows_per_file = {}
-
-    for part in parts:
-        parquet_file = pq.ParquetFile(str(part))
-        rows_per_file[part] = parquet_file.metadata.num_rows
 
-    return [(file, num_rows) for file, num_rows in rows_per_file.items()]
\ No newline at end of file
+def parquet_schema_to_pixeltable_schema(parquet_path: str) -> Dict[str, Optional[ts.ColumnType]]:
+    """Generate a default pixeltable schema for the given parquet file. Returns None for unknown types."""
+
+    input_path = Path(parquet_path).expanduser()
+    parquet_dataset = pa.parquet.ParquetDataset(input_path)
+    return to_pixeltable_schema(parquet_dataset.schema)
+
+
+def import_parquet(
+    cl: 'pixeltable.Client',
+    *,
+    path_str: str,
```

Comment:
I gather this is a table name? Can we come up with a more descriptive parameter name for it?